generator/templates/server/server.gotmpl

```diff
@@ -562,3 +597,33 @@ func (s *Server) TLSListener() (net.Listener, error) {
 	}
 	return s.httpsServerL, nil
 }
+
+func handleInterrupt(once *sync.Once, s *Server, quitting chan struct{}) {
+    once.Do(func(){
+	for _ = range s.interrupt {
+		if s.interrupted {
+			s.Logf("Server already shutting down")
+			continue
+		}
+		s.interrupted = true
+		s.Logf("Shutting down... ")
+		close(quitting)
+
+		//s.SetKeepAlivesEnabled(false)
+		if err := s.httpServerL.Close(); err != nil {
+			s.Logf("Error: %s", err)
+		}
+		if err := s.httpsServerL.Close(); err != nil {
+			s.Logf("Error: %s", err)
+		}
+		if err := s.domainSocketL.Close(); err != nil {
+			s.Logf("Error: %s", err)
+		}
+		//s.shutdown <- struct{}{}
```

Comment:
please remove commented line

Reply:
Sorry for the stray commented lines - will remove them and re-run generator.